test/common/index.js

```diff
@@ -246,7 +246,7 @@ Object.defineProperty(exports, 'opensslCli', {get: function() {
 
 Object.defineProperty(exports, 'hasCrypto', {
   get: function() {
-    return process.versions.openssl ? true : false;
+    return Boolean(process.versions.openssl);
```

Comment:
I think the usual way to do this in the project would be:  `!!process.versions.openssl`